CONTRIBUTING.md

```diff
@@ -82,19 +82,12 @@ the pull request affects. Valid areas as:
   - *Consensus* for changes to consensus critical code
   - *Docs* for changes to the documentation
   - *Qt* for changes to bitcoin-qt
+  - *Log* Changes to log messages
```

Comment:
Would say "Logging" or "Logs" to be consistent with "Mining" below or "Docs" above. "Log" isn't a component of bitcoin that you'd refer to with a singular noun.

Reply:
I don't disagree, yet suggested the opposite for brevity and what is often seen in practice.